From: Dario Faggioli <dfaggioli@suse.com>
Date: Tue, 20 Sep 2022 09:10:59 +0200
Subject: meson: remove $pkgversion from CONFIG_STAMP input to broaden
 compatibility

Git-commit: 0000000000000000000000000000000000000000

As part of the effort to close the gap with Leap I think we are fine
removing the $pkgversion component to creating a unique CONFIG_STAMP.
This stamp is only used in creating a unique symbol used in ensuring the
dynamically loaded modules correspond correctly to the loading qemu.
The default inputs to producing this unique symbol are somewhat reasonable
as a generic mechanism, but specific packaging and maintenance practices
might require the default to be modified for best use. This is an example
of that.

Signed-off-by: Bruce Rogers <brogers@suse.com>
Signed-off-by: Dario Faggioli <dfaggioli@suse.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 20fddbd707c91c1baee604202fb1..41f693a822a7ec58085bb53a8518 100644
--- a/meson.build
+++ b/meson.build
@@ -1736,7 +1736,7 @@ config_host_data.set_quoted('CONFIG_SYSCONFDIR', get_option('prefix') / get_opti
 if config_host.has_key('CONFIG_MODULES')
   config_host_data.set('CONFIG_STAMP', run_command(
       meson.current_source_dir() / 'scripts/qemu-stamp.py',
-      meson.project_version(), get_option('pkgversion'), '--',
+      meson.project_version(), '--',
       meson.current_source_dir() / 'configure',
       capture: true, check: true).stdout().strip())
 endif
